spec/rubocop/cop/rspec/empty_line_after_final_let_spec.rb

```diff
@@ -134,7 +137,7 @@
     RUBY
   end
 
-  it 'ignores empty lines between the lets' do
```

Comment:
The point of this example is different. I believe the best way is to add a newline between `let` and `it` and use "does not register an offence for empty lines between the lets".